src/styles/default.css

```diff
@@ -9,7 +9,6 @@
 */
 
 .hljs {
-  display: block;
   overflow-x: auto;
   padding: 1em;
```

Comment:
The padding on these is quite aggressive when applied to inline elements: 
![Screenshot from 2021-05-07 05-43-20](https://user-images.githubusercontent.com/2776014/117439175-21f8ad80-aef8-11eb-8bfb-aea936023518.png)

Should that be popped out onto its own rule for "code blocks"?

```diff
 .hljs {
   overflow-x: auto;
-  padding: 1em;
   background: #F0F0F0;
   color: #444;
 }
 
+pre .hljs {
+  padding: 1em;
+}
```

Doing so yields the following result:

![Screenshot from 2021-05-07 05-55-36](https://user-images.githubusercontent.com/2776014/117439790-e8747200-aef8-11eb-9e15-7f2805a51724.png)

Reply:
I thought you were done - I just picked it up and ran with it.  Does it look like I covered things?

Reply:
Yes, that looks alright.